### PR TITLE
Update index.html.markdown

### PR DESCRIPTION
**Pull request by epelkey**

### CODE REVIEW

File: website/docs/index.html.markdown [hunk 1]
```diff
@@ -43,7 +43,7 @@ The following arguments are supported:
 * `enable_ddi` - (Optional) This sets the permission schema to a DDI-compatible schema. 
 Users of the managed SaaS product should not need to set this.
 Users of DDI should set this to true if managing teams, users, or API keys through this provider.
-* `rate_limit_parallelism` - (Optional) Integer for parallelism amount (terraform's default is 10).
+* `rate_limit_parallelism` - (Optional) Integer for alternative rate limit and parallelism strategy.
```
Comment (mburtless) [inline website/docs/index.html.markdown:46]: Any reason to remove the parenthetical regarding the default value?

Comment (epelkey) [reply website/docs/index.html.markdown:46]: Oh, perhaps not. I must have cut it out while copying James' suggested edit. Am I able to adjust my suggested changes or should I submit another PR?

File: website/docs/index.html.markdown [hunk 2]
```diff
@@ -43,7 +43,7 @@ The following arguments are supported:
 * `enable_ddi` - (Optional) This sets the permission schema to a DDI-compatible schema. 
 Users of the managed SaaS product should not need to set this.
 Users of DDI should set this to true if managing teams, users, or API keys through this provider.
-* `rate_limit_parallelism` - (Optional) Integer for parallelism amount (terraform's default is 10).
+* `rate_limit_parallelism` - (Optional) Integer for alternative rate limit and parallelism strategy. (Terraform default value is 10.)
```
Comment (mburtless) [inline website/docs/index.html.markdown:46]: Typo?  Period should be outside of parenthetical

Comment (epelkey) [reply website/docs/index.html.markdown:46]: ugh. moving too quickly. good catch. fixing now. sorry!

Comment (epelkey) [reply website/docs/index.html.markdown:46]: oh wait not that's correct. The whole sentence is inside the parenthesis.

**Pull request merged**